packages/purpose-process/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist",
   "type": "module",
   "scripts": {
-    "test": "vitest",
+    "test": "vitest --config vitest.integration.config.ts",
```

Comment:
I changed the definition but did not rename the script to `test:integration` because this PR targets `develop` where https://github.com/pagopa/interop-be-monorepo/pull/1645 that modifies `turbo.json` is not merged yet. I think we can rename this to `test:integration` in the future PR where we'll add `test:api`. What do you think?